.circleci/scripts/binary_populate_env.sh

```diff
@@ -58,8 +58,7 @@ fi
 PIP_UPLOAD_FOLDER='nightly/'
 # We put this here so that OVERRIDE_PACKAGE_VERSION below can read from it
 export DATE="$(date -u +%Y%m%d)"
-#TODO: We should be pulling semver version from the base version.txt
-BASE_BUILD_VERSION="2.2.0.dev$DATE"
+BASE_BUILD_VERSION="$(cat ${PYTORCH_ROOT}/version|cut -da -f1).dev${DATE}"
```

Comment:
This should be version.txt I think

Reply:
Yeah, copy-n-paste from local to UI are hard :P